code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -47,11 +47,13 @@
 
 	to_chat(C, "[info_text]")
 
-	C.real_name = "Nightmare"
-	C.name = "Nightmare"
+	var/list/nightmarename = list("U'ruan", "Y`shej", "Nex", "Hel-uae", "Noaey'gief", "Mii`mahza", "Amerziox", "Gyrg-mylin", "Kanet'pruunance", "Vigistaezian")

```

Comment:
Instead of this use a .txt file, here's an example: https://github.com/tgstation/tgstation/blob/022174326edce19759e5677a715920212480cf70/code/_globalvars/lists/names.dm#L6
https://github.com/tgstation/tgstation/blob/022174326edce19759e5677a715920212480cf70/strings/names/death_commando.txt#L1-L70

Reply:
ok fine >:(